test/dd/test_dd_functionality.cpp

```diff
@@ -64,7 +64,7 @@ TEST_P(DDFunctionality, standardOpBuildInverseBuild) {
   using namespace qc::literals;
   auto gate = static_cast<qc::OpType>(GetParam());
 
-  qc::StandardOperation op;
+  qc::StandardOperation op, op_neg;
```

Comment:
First of all, you can see some of the clang-tidy warnings here in the "Files Changed" tab.
These should be easy to address.
In this case here, you do not seem to be using `op_neg` for the test at all, so it could be removed entirely.

Reply:
Done 👍🏼